### DISCUSSION TITLE
Funding / Monetization of the application.

### DISCUSSION THREAD

**Discussion by buildwithmalik** [AbdulMalikDev/ounotes#34] [Q&A]
### We're facing a problem

OU Notes requires an amount of money to be hosted. Firebase Database bills us every month an amount of money we have to pay according to the users on the platform. More specifically it charges us based on the Database Reads of the users. But we haven't monetized the application yet, so any ideas you guys have for monetization ?

**Comment by AbhiramRavula** [comment]
Add minimal ads when the application is in home page
